1657389 Add descriptions for SEO (#19543)
dotnet/docs: docs/csharp/programming-guide/classes-and-structs/constants.md

---
title: "Constants - C# Programming Guide"

> description: Constants in C# are immutable values, known at compile time, which do not change for the life of the program. Only C# built-in types can be constants.

Reviewer: Constants are a compile time construct, so I'm suggesting a change here:

```suggestion
description: Constants in C# are compile-time literal values, which do not change once the program is compiled. Only C# built-in types can be constants.
```
